Source/Eto/Drawing/PaddingConverter.cs

```diff
@@ -12,6 +12,11 @@ namespace Eto.Drawing
 	public class PaddingConverter : TypeConverter
 	{
 		/// <summary>
+		/// The character to split up the string which will be converted
+		/// </summary>
+		private readonly string[] StringSplitter = new string[1] { "," };
```

Comment:
The impact of a string[1] against a char[1] is in such a way marginal that it's hard to measure. To count a time: ~2ms for 1.000.000 iterations.
The reason in .NET is that string is not a type like char or int, it's a referenced type like an object or class. So it's referencing to char with some special implementations (e.g. comparisons). But even memory footprint should be nearly the same (16bit for char) in this case.
However I decided to changed the type to char like @firda-cze supposed.